main.go

```diff
@@ -2,17 +2,18 @@ package main
 
 import (
 	"context"
-	"github.com/cloudflare/tableflip"
-	"github.com/kailt/imageresizer/api"
-	"github.com/kailt/imageresizer/imager"
-	"github.com/spf13/viper"
 	"log"
 	"net/http"
 	"os"
 	"os/signal"
 	"strings"
 	"syscall"
 	"time"
+
```

Comment:
Cosmetic change unrelated to the fsnotify feature. Please revert. This file was already gofmt'ed.